platform/darwin/src/http_file_source.mm

```diff
@@ -231,6 +233,8 @@ void cancel() {
                 using Error = Response::Error;
 
                 if (error) {
+                    MGLLogError(@"Requesting: %@ failed with error: %@", res.URL.relativePath, error);
```

Comment:
We should look at Activity Tracing for methods like this. Beyond the scope of this PR, but something to consider later.